ui/src/views/Hooks/ListHooks/index.jsx

```diff
@@ -51,19 +54,16 @@ export default class ListHooks extends Component {
       description,
       data: { loading, error, hookGroups },
     } = this.props;
-    const query = qs.parse(this.props.location.search.slice(1));
+    const query = parse(this.props.location.search.slice(1));
     const hookSearch = query.search;
-    const tree = hookGroups
-      ? hookGroups.map(group => ({
-          value: group.hookGroupId,
-          nodes: group.hooks.map(hook => ({
-            value: hook.hookId,
-            href: `/hooks/${group.hookGroupId}/${encodeURIComponent(
-              hook.hookId
-            )}`,
-          })),
-        }))
-      : [];
+
+    let count = 0;
+
+    const renderTree = (nodes) => (nodes.map(
+      <TreeItem key={count++} nodeId={count} label={nodes.hookGroupId}>
```

Comment:
`nodes.map` [takes a callback function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) as the first argument but is currently being fed a React element. That probably explains why it's acting weird. Hope that helps.